### PR TITLE
"What we learned" slides

### PR DESCRIPTION
**Pull request by varya**

### CODE REVIEW

File: slides/ds101-worskhop.mdx [hunk 1]
```diff
@@ -127,12 +127,21 @@ We had a lot of fun!
...
+# Review
 
 </Primary>
 
 ---
 
+## What we learned making toasts
+
+- Got to know each other
+- Worked in Zoom rooms
```
Comment (okonet) [inline slides/ds101-worskhop.mdx]: Let’s not use past sense here

Comment (varya) [reply slides/ds101-worskhop.mdx]: @okonet I fixed wording in this and one more piece.

**Review by okonet — APPROVED** [review]

**Comment by okonet** [comment]
Conflicts :(

File: slides/ds101-worskhop.mdx [hunk 2]
```diff
@@ -425,6 +448,23 @@ to find out the relevant parts of your design system.
 
 ---
 
+<Figure
+  src="/assets/parts-nathan-sheet.png"
+  alt="A Team Activity to Start a Design System"
+/>
+
+[Picking Parts, Products & People by Nathan Curtis](https://medium.com/eightshapes-llc/picking-parts-products-people-a06721e81742)
```
Comment (okonet) [inline slides/ds101-worskhop.mdx:456]: Add this as a caption to `Figure`?

**Review by okonet — APPROVED** [review]

**Pull request merged**